WeChat/CustomViewCOntroller.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CustomViewCOntroller.swift
```

Comment:
Rename this file name `CustomViewCOntroller -> CustomViewController` Or something better that makes sense.